### DISCUSSION TITLE
Dispositivo Zigbee perde connessione

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2137] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti,
sto iniziando a configurare i dispositivi Bticino tramite Sonoff dongle, se lo accoppio viene rilevato, funziona correttamente ma dopo alcuni giorni perde la connessione.
lo elimino, lo riassocio ma dopo 2/3gg stessa cosa.
ora sto provando con i K4027C (tapparelle).

**Comment by ster1um** [comment]
> Ciao a tutti, sto iniziando a configurare i dispositivi Bticino tramite Sonoff dongle, se lo accoppio viene rilevato, funziona correttamente ma dopo alcuni giorni perde la connessione. lo elimino, lo riassocio ma dopo 2/3gg stessa cosa. ora sto provando con i K4027C (tapparelle).

Hai provato a cambiare canale zigbee

**Comment by Xabar21** [comment]
ho lo stesso identico problema con due sensori per le finestre ed un termometro
non penso sia una questione di frequenze perchè la wifi 2.4 l'ho impostata sul canale 1, quella zigbee sull'11
seguo, nel caso in cui qualcuno avesse soluzioni

**Reply by snakuzzo** [reply]
Non so se i vostri siano problemi legati ad interferenze, ma guarda che i canali zigbee e wifi non è che sono proprio uguali :D
Non è che il canale 1 del wifi sia uguale al canale 1 zigbee.
Guarda [QUI](https://www.metageek.com/training/resources/zigbee-wifi-coexistence/)

**Reply by Xabar21** [reply]
utilissimo. grazie

**Reply by ster1um** [reply]
> ho lo stesso identico problema con due sensori per le finestre ed un termometro non penso sia una questione di frequenze perchè la wifi 2.4 l'ho impostata sul canale 1, quella zigbee sull'11 seguo, nel caso in cui qualcuno avesse soluzioni

Io consiglio sempre il canale 25 per zigbee.
Se vuoi approfondire cerca su Google "zigbee channel coexistence "

**Reply by snakuzzo** [reply]
...che è la stessa cosa che gli avevo già linkato io :)

**Reply by ster1um** [reply]
> ...che è la stessa cosa che gli avevo già linkato io :)

Scusa non ho aperto il tuo link... Pardon

**Reply by icemanno** [reply]
Ciao a tutti scusate, mi servirebbe un info non pertinente alla connessione, avendo casa su più piani posso installare più di uno usb zigbee ?
Grazie

**Comment by 0p3t0** [comment]
Ciao, anche io ho lo stesso problema su alcuni elementi zigbee.
Ho Conbee-II come antenna, e sono abbastanza certo che non sia un problema di portata in quanto ho un sensore molto lontano che comunica correttamente, mentre ho un pulsante (ikea), che si disconnette, così come altri sensori aqara discretamente vicini al Conbee-II.
Sarei interessato a capire se c'è qualche modo per individuare il problema, avete qualche suggerimento?
Grazie

**Reply by snakuzzo** [reply]
A parte le prove, non c'è altro che tu possa fare.
Piccola nota...tutti i dispositivi alimentati a batteria vanno in deep sleep e si disconnecttono. E' normale che il tuo telecomandino ed i sensori siano disconnessi. Si riattivano in automatico per qualche secondo ogni tot di tempo e quando c'è un cambio stato.
Se fossero sempre connessi, quelle batterie durerebbero un paio di giorni.

**Reply by 0p3t0** [reply]
Questo mi è chiaro, ma il fatto è che anche premendo il pulsante, guardando i log, ed interagendoci non diano segni di vita.
Facendo un esempio pratico: Ho un sensore di temperatura/umidità aquara in cucina che comunica correttamente, e quando premo il tastino, invia i dati, ed un'altro all'ingresso, ad un paio di metri da quello in cucina, e quest'ultimo non comunica correttamente: Se tengo premuto il pulsante per 5 secondi, si riannuncia, e riprende a comunicare per qualche ora/giorno, e successivamente smette.

**Reply by ster1um** [reply]
Uso anch'io dei remote Ikea senza grossi problemi.
Quando si disconnettono sono utilizzati da giorni?
Hai altri elementi zigbee connessi alla rete elettrica?

**Reply by 0p3t0** [reply]
Il telecomando in questione
 ( ![image](https://user-images.githubusercontent.com/59409205/208311531-0339724a-f178-49b8-8122-dbc527dba4f2.png) 
)viene utilizzato tutti i giorni, ma da un giorno all'altro mi si scollega.
Mi gestisce una lampadina philips (che fa da repeater), ed ho questo componente per i consumi, collegato alla corrente

![image](https://user-images.githubusercontent.com/59409205/208311559-2e425b16-bb28-4884-96e1-6f9ae7e31146.png),
il resto batteria

Ho anche un'altro telecomando ikea identico e funziona, ed altri interruttori ikea (on off) che non danno problemi, così come alcuni altri sensori aqara che funzionano sempre.
Al momento l'ho sostituito e gli faccio fare altro (magari è guasto lui), ma la cosa che mi preme capire sono le disconnessioni dei sensori aqara, resettato e riconnesso stamattina un sensore, ha comunicato per qualche ora e poi stop.
![image](https://user-images.githubusercontent.com/59409205/208312069-4a731515-306d-411d-b0b3-dea906a45a54.png)

**Reply by ster1um** [reply]
> Il telecomando in questione ( ![image](https://user-images.githubusercontent.com/59409205/208311531-0339724a-f178-49b8-8122-dbc527dba4f2.png) )viene utilizzato tutti i giorni, ma da un giorno all'altro mi si scollega. Mi gestisce una lampadina philips (che fa da repeater), ed ho questo componente per i consumi, collegato alla corrente
> 
> ![image](https://user-images.githubusercontent.com/59409205/208311559-2e425b16-bb28-4884-96e1-6f9ae7e31146.png), il resto batteria
> 
> Ho anche un'altro telecomando ikea identico e funziona, ed altri interruttori ikea (on off) che non danno problemi, così come alcuni altri sensori aqara che funzionano sempre. Al momento l'ho sostituito e gli faccio fare altro (magari è guasto lui), ma la cosa che mi preme capire sono le disconnessioni dei sensori aqara, resettato e riconnesso stamattina un sensore, ha comunicato per qualche ora e poi stop. ![image](https://user-images.githubusercontent.com/59409205/208312069-4a731515-306d-411d-b0b3-dea906a45a54.png)

Visto che la rete non ha molti dispositivi non puoi passare al canale 25 e vedere se fa ancora casino?

**Reply by 0p3t0** [reply]
Ho modificato il canale sul 14 dopo avere sistemato gli access point di casa, ora provo a vedere qualche giorno cosi', poi sposto sul 25 per vedere se migliora, intanto grazie :)
PS: I dispositivi zigbee al momento sono 24.

**Reply by 0p3t0** [reply]
Aggiornamento: dopo qualche giorno stesso problema: qui si vede la linea verde relativa al sensore aqara zigbee, e la linea galla relativa alla temperatura netatmo.
i due valori che generano la linea sono relativi al riavvio, in quanto sembra mantenere l'ultimo valore comunicato.

![image](https://user-images.githubusercontent.com/59409205/210979765-2847bf83-5ff5-4b64-ab17-94d6fa04435f.png)

**Comment by diaz66** [comment]
Utilizzo ZigBee2mqtt da poco e anche a me capita con dispositivi a batteria tipo sensore temperatura. Ho fatto varie prove ed anche se non centra nulla, ho **disattivato il controllo disponibilità.**  E adesso sono un pò di giorni che non perde un colpo.....mah....mi aspetto che prima o poi....
![image](https://user-images.githubusercontent.com/32227980/213286898-6f9f5b33-7c93-4576-aa4c-45c4e2e12226.png)
![image](https://user-images.githubusercontent.com/32227980/213287106-193a8e45-646d-4efb-8577-4084e845e05d.png)
![image](https://user-images.githubusercontent.com/32227980/213288396-eb63e574-96dc-4a3f-b2f8-b43a9e307cf2.png)

**Comment by faustozago** [comment]
Ciao a tutti,
anch'io sto riscontrando lo stesso problema...ma non con tutti i dispositivi.
Ho dongle sonoff plus V2 , ho 3 dispositivi sonoff (temperatura, sensore porta, sensore temperatura ) che non perdono un colpo.
Ora sto provando 2 sensori di movimento VISONIC MP-840 (compatibili con Z2M) , che vengono riconosciuti e accoppiati subito , e funzionano benissimo , ma quando riavvio HA o Z2M , i sensori non funzionano piu , rimangono nella lista dispositivi con  un numero di ricezione alto , appaiono anche nella mappa ma  sopra la linea di collegamento appare il numoro 0 .

Qualcuno ha trovato la soluzione a questo problema?

**Comment by developerGM** [comment]
Ciao a tutti,
mi collego a questo thread perchè ho lo stesso problema con alcuni dispositivi a batteria. In particolare si tratta dei dispositivi Aqara anti allagamento. Per 2 giorni stanno collegati e li vedo anche in mappa, poi si scollegano.
Qualcuno ha trovato soluzioni?
Mi era stato detto di mettere pile solo Duracell, ma stesso risultato

**Comment by trebestie** [comment]
Il problema è il Sonoff E. 
Per risolvere lo avevo sostituito con la versione precedente.
